src/index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
-import App from './components/app';
+import App from './components/app.jsx';
```

Comment:
It is not required to use `.jsx` file extensions, but if you really want to it is okay.